pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -1384,13 +1384,13 @@ public void classInJenkinsfile() throws Exception {
     @Issue("JENKINS-47109")
     @Test
     public void parallelStagesFailFast() throws Exception {
-        expect(Result.ABORTED, "parallelStagesFailFast")
+        expect(Result.FAILURE, "parallelStagesFailFast")
```

Comment:
I haven't added any new tests since the modifications needed here and in `parallelStagesFailFastWithOption` are the behavior we are testing. Might be good to add some some more complicated tests involving post conditions and nested stages. Should also add another `@Issue` for the current ticket.

Reply:
There's probably existing tests that could be similarly modified. Heck, there's probably a bunch of them that'll need tweaking (or even consolidation?) anyway.